README.md

```diff
@@ -298,6 +300,31 @@ order to use EL function mapping, the folder `o2a_libs` should
 be copied over to the Airflow DAG folder. This should then be picked up and
 parsed by the Airflow workers and then available to all DAGs.
 
+## Workflow and node notifications
+
+Workflow jobs can be configured to make an HTTP GET notification upon start and end of a workflow action node
+and upon the start and completion of a workflow job. More information in [Oozie docs](https://oozie.apache.org/docs/5.1.0/WorkflowFunctionalSpec.html#a5_Workflow_Notifications).
+
+Oozie-to-Airflow supports this feature.
+The `job.properties` file has contain URLs for workflow and action node notifications - example below:
+
+```
+oozie.wf.workflow.notification.url=http://example.com/workflow?job-id=$jobId&status=$status
+oozie.wf.action.notification.url=http://example.com/action?job-id=$jobId&node-name=$nodeName&status=$status
+```
+
+If they are present, Oozie-to-Airflow will insert additional `HttpOperator` to the generated DAG
```

Comment:
`HttpOperator`?  In my implementation, I use a BashOperator. Has this changed?

Reply:
Right, my bad! Thanks, fixed in c9a1b0d09ece704529370db58f39966bb6bffd07